CMakeLists.txt

```
@@ -271,6 +271,16 @@ if(CATKIN_ENABLE_TESTING)
     ${catkin_LIBRARIES}
   )
 
+  ########################
+  ##  Acceptance-Tests  ##
+  ########################
+  add_executable(acceptancetest_publish_test
+    test/acceptance_tests/acceptancetest_publish_test.cpp
+  )
+  target_link_libraries(acceptancetest_publish_test
+    ${catkin_LIBRARIES}
+  )
```

```suggestion
if(ENABLE_HW_TESTS)
  add_executable(acceptancetest_publish_test
    test/acceptance_tests/acceptancetest_publish_test.cpp
  )
  target_link_libraries(acceptancetest_publish_test
    ${catkin_LIBRARIES}
  )
endif()
```

otherwise travis is red